gossip/emitter/sync.go

```diff
@@ -51,8 +51,7 @@ func (em *Emitter) currentSyncStatus() doublesign.SyncStatus {
 	if em.world.IsSynced() {
 		s.P2PSynced = em.syncStatus.p2pSynced
 	}
-	prevEmitted := em.readLastEmittedEventID()
-	if prevEmitted != nil && (em.world.GetEvent(*prevEmitted) == nil && em.epoch <= prevEmitted.Epoch()) {
+	if prevEmitted := em.readLastEmittedEventID(); prevEmitted != nil && (em.world.GetEvent(*prevEmitted) == nil && em.epoch <= prevEmitted.Epoch()) {
```

Comment:
Hmm, this one became too long

Reply:
Does multiline if statement sound good to you?  It does not make sense to declare `prevEmitted` as local in `currentSyncStatus` since it is used only inside if statement.
```
if prevEmitted := em.readLastEmittedEventID(); prevEmitted != nil &&
		em.world.GetEvent(*prevEmitted) == nil &&
		em.epoch <= prevEmitted.Epoch() {
		s.P2PSynced = time.Time{}
	}
```

Reply:
```It does not make sense to declare prevEmitted as local```
Makes sense! The last variant looks good